client/src/utils/interfaces/workersTable.ts

```diff
@@ -0,0 +1,54 @@
+import { ReactElement } from 'react';
+
+export interface Column {
+  id: (
+    'name'
+    | 'email'
+    | 'gender'
+    | 'location'
+    | 'phone'
+    | 'dateOfBirth'
+    | 'state'
+    | 'specialty'
+    | 'yearsOfExperience'
+    | 'hiringDate'
+    | 'joiningDate'
+    | 'drawer'
+  );
```

Comment:
you can do `keyof Employee` or `keyof Pick<Employee, 'name'
    | 'email'
    | 'gender'
    | 'location'
    | 'phone'
    | 'dateOfBirth'
    | 'state'
    | 'specialty'
    | 'yearsOfExperience'
    | 'hiringDate'
    | 'joiningDate'>`